services-api/src/test/java/io/scalecube/services/methods/ServiceMethodInvokerTest.java

```diff
@@ -152,8 +160,10 @@ void testInvokeOneWhenThrowException() throws Exception {
             false,
             AUTH);
 
+    // FIXME: 19.02.2020
```

Comment:
whats needs to be fixed here?

Reply:
Outdated changes. I used to think of doing a Supplier<*ServiceInstance* instead of a service instance in ServiceInfo to get a service instance lazy. Otherwise, you couldn't inject ServiceDiscovery because it hadn't yet been initialized when the service instance was created.